devito/petsc/solve.py

```diff
@@ -7,6 +7,7 @@
 from devito.petsc.types import PETScArray, LinearSolveExpr, MatVecEq, RHSEq
 
 from sympy import simplify
```

Comment:
Nitpick: line break between sympy/numpy/etc imports and Devito imports. Also group Devito imports